src/kixi/hecuba/onyx/jobs/hecuba_fetcher.clj

```diff
@@ -1,5 +1,36 @@
-(ns kixi.hecuba.onyx.jobs.hecuba-fetcher)
+(ns kixi.hecuba.onyx.jobs.hecuba-fetcher
+  (:require [cheshire.core :as json]
+            [clj-http.client :as client]
+            [environ.core :refer [env]]))
+
+(def api-endpoint "http://localhost:8010/4/")
+
+(defn run-api-search [{:keys [entity-id] :as args-map}]
+  (let [url-to-get (str api-endpoint
+                        "entities/"
+                        entity-id
+                        "/devices/")]
+    (try (let [response-json (-> (:body (client/get
+                                         url-to-get
+                                         {:basic-auth [(env :hecuba-username)
+                                                       (env :hecuba-password)]
+                                          :Headers {"X-Api-Version" "2"}
+                                          :content-type :json
+                                          :socket-timeout 20000
+                                          :conn-timeout 20000}))
+                                 (json/parse-string))]
```

Comment:
```
(-> url-to-get
    (client/get ...) ;; maybe in another func?
    :body
    (json/parse-string)
```

And I wonder if we should use plumatic/schema to validate what is returned and return a type of vector [errors data]

wdyt?